sled-agent/src/bootstrap/early_networking.rs

```diff
@@ -646,17 +655,22 @@ impl<'a> EarlyNetworkSetup<'a> {
                 rq.routes.list.push(sr);
             }
         }
-        mgd.static_add_v4_route(&rq).await.map_err(|e| {
-            EarlyNetworkSetupError::BgpConfigurationError(format!(
-                "static routing configuration failed: {e}",
-            ))
-        })?;
+
+        if let Err(e) = mgd.static_add_v4_route(&rq).await {
+            error!(
+                self.log,
+                "BGP peer configuration failed";
```

Comment:
This message should be about static routing, not BGP.

Reply:
Yay for copy paste errors